recipes/scikit-build/python27-x64.patch

```diff
@@ -0,0 +1,13 @@
+MSVC 9 x64 is broken on appveyor. Use NMake there.
+See https://github.com/conda/conda-build/issues/1327
+
+diff --git a/skbuild/platform_specifics/windows.py b/skbuild/platform_specifics/windows.py
+index 44f1816..fcfd9dc 100644
+--- a/skbuild/platform_specifics/windows.py
++++ b/skbuild/platform_specifics/windows.py
+@@ -45,3 +45,4 @@ class WindowsPlatform(abstract.CMakePlatform):
+         # string IDs seem to be just the vs_base.
+ 
+         self.default_generators.insert(0, vs_base)
++        self.default_generators.insert(1, "NMake Makefiles")
```

Comment:
@jcfr, I've added this patch because of this issue, https://github.com/conda/conda-build/issues/1327